engine/src/main/java/io/graphqlcrud/SQLQueryBuilderVisitor.java

```diff
@@ -271,67 +273,130 @@ public void visitArgument(Field field, GraphQLFieldDefinition definition, GraphQ
+            case "ge":
+                c = left.ge(((IntValue) v).getValue());
+                break;
+        }
         return c;
     }
 
     public Condition visitFloatValue(Value v, org.jooq.Field left, String conditionName) {
         Condition c = null;
-        if(conditionName.equals("eq"))
-            c = left.eq(((FloatValue)v).getValue());
-        else if(conditionName.equals("ne"))
-            c = left.ne(((FloatValue) v).getValue());
-        else if(conditionName.equals("lt"))
-            c = left.lt(((FloatValue) v).getValue());
-        else if(conditionName.equals("le"))
-            c = left.le(((FloatValue) v).getValue());
-        else if(conditionName.equals("gt"))
-            c = left.gt(((FloatValue) v).getValue());
-        else if(conditionName.equals("ge"))
-            c = left.ge(((FloatValue) v).getValue());
+        switch (conditionName) {
+            case "eq":
+                c = left.eq(((FloatValue) v).getValue());
+                break;
+            case "ne":
+                c = left.ne(((FloatValue) v).getValue());
+                break;
+            case "lt":
+                c = left.lt(((FloatValue) v).getValue());
+                break;
+            case "le":
+                c = left.le(((FloatValue) v).getValue());
+                break;
+            case "gt":
+                c = left.gt(((FloatValue) v).getValue());
+                break;
+            case "ge":
+                c = left.ge(((FloatValue) v).getValue());
+                break;
+        }
+        return c;
+    }
+
+
+    public Condition visitArrayValue(List<Value> v, org.jooq.Field left, String conditionName) {
+        Condition c = null;
+        switch (conditionName) {
+            case "between":
+                if (v.get(0) instanceof FloatValue)
+                    c = left.between(((FloatValue) v.get(0)).getValue(), ((FloatValue) v.get(1)).getValue());
+                else if (v.get(0) instanceof IntValue)
+                    c = left.between(((IntValue) v.get(0)).getValue(), ((IntValue) v.get(1)).getValue());
+                break;
+            case "in":
+                if (v.get(0) instanceof StringValue) {
+                    List<String> list = new ArrayList<>();
+                    for (Value value : v)
+                        list.add(((StringValue) value).getValue());
+                    c = left.in(list);
+                } else if (v.get(0) instanceof FloatValue) {
+                    List<BigDecimal> list = new ArrayList<>();
+                    for (Value value : v)
+                        list.add(((FloatValue) value).getValue());
+                    c = left.in(list);
+                } else if (v.get(0) instanceof IntValue) {
+                    List<BigInteger> list = new ArrayList<>();
+                    for (Value value : v)
+                        list.add(((IntValue) value).getValue());
+                    c = left.in(list);
+                }
```

Comment:
put in a `default` in here and throw a `runtimeexception` saying wrong condition